drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/create_referral_job.rb

```diff
@@ -5,7 +5,8 @@
 ###
 
 module HmisExternalApis::AcHmis
-  class CreateReferralJob < ApplicationJob
+  class CreateReferralJob < BaseJob
+    queue_as ENV.fetch('DJ_SHORT_QUEUE_NAME', :short_running)
```

Comment:
This actually doesn't get queued, we always call it with perform_now. But fine to add this, short_running makes sense